trackingC_common.R

```diff
@@ -16,3 +21,18 @@ if(Sys.getenv("CI") == "true") {
 
 # Set range of years for output data
 OUTPUT_YEARS <- 1950:2200
+
+# Define name and units of all parameters
+name_vector <- c("BETA" = BETA(), "Q10_RH" = Q10_RH(), 
+                 "ECS" = ECS(), "NPP_FLUX0" = NPP_FLUX0(),
+                 "AERO_SCALE" = AERO_SCALE(), "DIFFUSIVITY" = DIFFUSIVITY(),
+                 "LUC_SCALAR" = "LUC_SCALAR"
+)
+
+units_vector <- c("BETA" = "(unitless)", "Q10_RH" = "(unitless)", 
+                  "ECS" = "degC", "NPP_FLUX0" = "Pg C/yr",
+                  "AERO_SCALE" = "(unitless)", "DIFFUSIVITY" = "cm2/s",
+                  "LUC_SCALAR" = "(unitless)"
+)
+
+scalar <- "LUC_SCALAR"
```

Comment:
`scalar` is very non-specific; why not `luc_scalar` or (better to my eyes, since this is a constant) `LUC_SCALAR`?

Reply:
I wasn't sure if we were going to introduce more scaling constants for anything - this would have been a vector of parameters that the run loop would check against. But if we're just sticking with LUC alone, I can change or just modify the loop with the name vector.

Reply:
I think we will definitely want to do some visualizations, so this may be the easiest way.

Reply:
Oh interesting idea about multiple scalars. I don't know. Hmm!